packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -673,6 +673,115 @@ storiesOf('TableView', module)
       </TableView>
     )
   )
+  .add(
+    'should fill cell width allowsResizing',
+    () => (
+      <TableView aria-label="TableView with filled cells" selectionMode="multiple" width={500} height={200} onSelectionChange={s => onSelectionChange([...s])}>
+        <TableHeader>
+          <Column allowsResizing>File Name</Column>
+          <Column align="center" allowsResizing>Type</Column>
+          <Column align="end" allowsResizing>Size</Column>
```

Comment:
Noticed that the menus doesn't quite align properly when the columns are expanded
<img width="497" alt="image" src="https://user-images.githubusercontent.com/9661127/212399821-5a9b7227-6eae-4423-98eb-9f370057e16f.png">

Already happens in main, but not in [this build from Dec: ](https://github.com/adobe/react-spectrum/commit/ca8f4230c5efb9f248f0a028257ac60093706f78. I'll do some more digging to see when this changed, not too bothered by it tbh

Reply:
Yeah, this is due to the issue that the contributor is trying to fix. Once that gets resolved, this will get fixed